### PR TITLE
Creation of atan.md for Golang

### PR DESCRIPTION
**Pull request by arisdelacruz**
### Description

This is my attempt to create the resource for the Atan() function in Golang.

### Type of Change

- Adding a new entry

### Checklist

- [ / ] All writings are my own.
- [ / ] My entry follows the Codecademy Docs style guide.
- [ / ] My changes generate no new warnings.
- [ / ] I have performed a self-review of my own writing and code.
- [ / ] I have checked my entry and corrected any misspellings.
- [ / ] I have made corresponding changes to the documentation if needed.
- [ / ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ / ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ / ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/go/concepts/math-functions/terms/atan/atan.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+Tags:
+  - 'Math'
+  - 'Trigonometry'
+  - 'Functions'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`Atan()`** function returns the arctangent of a value given in radians.
```
Comment (SSwiniarski) [inline content/go/concepts/math-functions/terms/atan/atan.md:16]:
```suggestion
The **`Atan()`** function returns the arctangent, in radians, of a given value.
```

File: content/go/concepts/math-functions/terms/atan/atan.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+## Syntax
+
+```pseudo
+result := math.Atan(x)
+```
+
+Notes on the parameter:
+
+- `x` is the value whose arctangent value is to be found.
+- The range must be between between -π/2 to π/2 radians.
```
Comment (SSwiniarski) [inline content/go/concepts/math-functions/terms/atan/atan.md:27]:
```suggestion
- The `result` will be between between -π/2 to π/2 radians.
```

File: content/go/concepts/math-functions/terms/atan/atan.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+- If the parameter is either less than -π/2 or greater than π/2 it returns NaN.
```
Comment (SSwiniarski) [inline content/go/concepts/math-functions/terms/atan/atan.md:28]:
```suggestion
- If the parameter is `NaN` it returns `NaN`.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@arisdelacruz I did a first review and left a couple of comments.

**Comment by arisdelacruz** [comment]
@SSwiniarski 

thanks

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@arisdelacruz Looks good for a second review.

**Review by SSwiniarski — APPROVED** [review]

File: content/go/concepts/math-functions/terms/atan/atan.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+The **`Atan()`** function returns the arctangent, in radians, of a given value.
+
+## Syntax
+
+```pseudo
+result := math.Atan(x)
+```
+
+Notes on the parameter:
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/atan/atan.md:24]:
```suggestion
Notes on `.Atan()`:
```

File: content/go/concepts/math-functions/terms/atan/atan.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+- `x` is the value whose arctangent value is to be found.
+- The `result` will be between between -π/2 to π/2 radians.
+- If the parameter is `NaN` it returns `NaN`.
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/atan/atan.md:28]:
```suggestion
- If the argument is `NaN` it returns `NaN`.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@arisdelacruz Just a couple of small additional edits. Please update, and tag me when you've completed the changes. Thanks

**Comment by arisdelacruz** [comment]
@caupolicandiaz 

done

thanks

**Pull request merged**